store/postgres/src/store.rs

```diff
@@ -1016,6 +967,21 @@ impl StoreTrait for Store {
         if block_ptr_from.number != block_ptr_to.number + 1 {
             panic!("revert_block_operations must revert a single block only");
         }
+        // Don't revert past a graft point
+        let info = self.subgraph_info(&subgraph_id)?;
+        if let Some(graft_block) = info.graft_block {
+            if graft_block as u64 > block_ptr_to.number {
+                return Err(format_err!(
+                    "Can not revert subgraph `{}` to block {} \
+                  as it was grafted at block {} and we do not allow reverting \
```

Comment:
Let's rephrase the `we do not allow` to `reverting past graft points is not allowed` or something else that does not involve "us".

Reply:
I changed it to `and reverting past a graft point is not possible`